<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CabecerasFits</class>
 <widget class="QWidget" name="CabecerasFits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="nombreProyectopushButton">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>0</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CM-RRLYRAE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nombre del Proyecto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="raHhLineEdit">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>36</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="raMmLineEdit">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>36</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="raSsLineEdit">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>36</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>36</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>36</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>36</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>(HH:MM:SS.SS)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Foco del Telescopio:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="focoTelescopioLineEdit">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>70</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Foco de Chequeado:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="siRadioButton">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>100</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Si</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="noRadioButto">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>100</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="anguloHorarioHhLineEdit">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>130</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="anguloHorarioMmLineEdit">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Angulo Horario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>(HH:MM:SS.SS)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>130</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="anguloHorarioSsLineEdit">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Declinacion:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="declinacionAnguloComboBox">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>170</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>(GG:MM:SS.SS)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="declinacionGgLineEdit">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Declinacion de Angulo Horario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="declinacionMmLineEdit">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="declinacionSsLineEdit">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreObjetoLineEdit">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>250</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nombre del Objeto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>290</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Posicion de los dedos: (x,y,z,t)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="posicionDedosXLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="posicionDedosYLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="posicionDedosZLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="posicionDedosTLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaDedosXLineEdit">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>330</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temperatura de los dedos: (x,y,z,t)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaDedosYLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaDedosZLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaDedosTLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vacio de la Camara:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vacioCamaralineEdit">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>370</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vacioLineaSuperiorLineEdit">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>410</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vacio de la Linea Superior:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vacioLineaInferiorLineEdit">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>450</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vacio de la Linea Inferior:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>490</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temperatura de la Nevera:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaNeveraLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="filtro1PushButton">
   <property name="geometry">
    <rect>
     <x>683</x>
     <y>-1</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>NO_SELECCIONADO</string>
   </property>
  </widget>
  <widget class="QLabel" name="filtro1Label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtro 1:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filtro2PushButton">
   <property name="geometry">
    <rect>
     <x>683</x>
     <y>30</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>NO_SELECCIONADO</string>
   </property>
  </widget>
  <widget class="QLabel" name="filtro2Label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>34</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtro 2:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filtro3PushButton">
   <property name="geometry">
    <rect>
     <x>683</x>
     <y>65</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>NO_SELECCIONADO</string>
   </property>
  </widget>
  <widget class="QLabel" name="filtro3Label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtro 3:</string>
   </property>
  </widget>
  <widget class="QLabel" name="filtro4Label">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>105</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtro 4:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filtro4PushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>NO_SELECCIONADO</string>
   </property>
  </widget>
  <widget class="QLabel" name="instrumentoUsadoLabel">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>147</y>
     <width>142</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Instrumento Usado:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="instrumentoUsadoComboBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>140</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filtro4Label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Orientacion del Primsa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>186</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Grados)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="orientacionPrismaLineEdit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="observadorComboBox">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>220</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="observadorLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>88</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Observador:</string>
   </property>
  </widget>
  <widget class="QLabel" name="condicionesDelCieloLabel">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>264</y>
     <width>159</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Condiciones del Cielo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="condicionesCieloComboBox">
   <property name="geometry">
    <rect>
     <x>694</x>
     <y>264</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Grados celcius)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaDomoLineEdit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>304</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filtro4Label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>304</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temperatura del domo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaDomoLineEdit_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>340</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>346</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="filtro4Label_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Humedad del domo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>386</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(DS-Modificado)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frecuenciaYicLineEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>380</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filtro4Label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Frecuencia YIC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="filtro4Label_6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Frecuencia RA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frecuenciaRaLineEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>416</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(DS-Modificado)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frecuenciaDecLineEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>444</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filtro4Label_7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>444</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Frecuencia DEC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>450</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frecuencia YIC:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cerrarPushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>480</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
